src/lib.rs

```diff
@@ -720,6 +720,16 @@ fn sanity_check_sdp_session(session: &SdpSession) -> Result<(), SdpParserError>
             }
         }
 
+        if msection
+            .get_attribute(SdpAttributeType::RtcpMuxOnly)
+            .is_some()
+            && msection.get_attribute(SdpAttributeType::RtcpMux).is_none()
+        {
+            return Err(make_seq_error(
+                "rtcp-mux-only requires to have a rtcp-mux attribute as well",
```

Comment:
I would like to propose the following rewording: `rtcp-mux-only media sections must also contain the rtcp-mux attribute`